tools/chouette/initialize-chouette.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+echo "1. Create /opt/chouette/ folder"
+if [ ! -d "/opt/chouette/" ]; then
+
+  sudo mkdir /opt/chouette
+  response=$?
+  if [ $response -eq 0 ]; then
+      echo "OK"
+  else
+      echo "FAIL"
+  fi
+
+else
+  echo "Directory /opt/chouette/ exists."
+fi
+
+echo "2. Copy chouette.zip to /opt/chouette"
+sudo cp mobi.chouette.command-3.4.8.zip /opt/chouette/
+if [ $? -eq 0 ]; then
+    echo "Copy - OK"
+else
+    echo "Copy - FAIL"
+fi
+
+echo "3. Unzip chouette.zip"
+sudo cd /opt/chouette
+sudo unzip /opt/chouette/mobi.chouette.command-3.4.8.zip -d /opt/chouette/
+
+echo "4. Give permissions "
+sudo cd ..
+sudo chown -R markusva:staff /opt/chouette
```

Comment:
Something alse than `markusva` here... :)
```suggestion
sudo chown -R markusva:staff /opt/chouette
```

If user can't be resolved then would `chmod ugo+rwx` be ok...?

Reply:
haha. thanx!
Fixed by getting current user instead of hardcoded value.